cmake/BuildAres.cmake

```diff
@@ -47,7 +47,8 @@ macro(ares_build)
         TMP_DIR ${ARES_BINARY_DIR}/tmp
         STAMP_DIR ${ARES_BINARY_DIR}/stamp
         BINARY_DIR ${ARES_BINARY_DIR}
-        CMAKE_ARGS ${ARES_CMAKE_FLAGS})
```

Comment:
After I've applied LuaJIT patch from this branch https://github.com/tarantool/luajit/tree/ligurio/ninja-support

I've received these timings on my rather slow WSL2 subsystem if build using 4 concurrent job-servers:

```
cmake .. -DCMAKE_BUILD_TYPE=Debug

time make -j4
...
real 4m14.999s
user 8m56.113s
sys 2m45.313s
...
real 4m7.887s
user 12m19.959s
sys 2m40.332s
```

but for Ninja they were as such:
```
time ninja -j4
real 3m4.345s
user 7m3.177s
sys 2m4.206s
...
real 2m17.384s
user 6m45.717s
sys 1m21.037s
```

Take into consideration that I've ran Ninja first, thus it's 3 minutes was actually a cold start, all the others (2nd Ninja invocation and both Make invocations were actually warm starts, with file cache warmed up).

i.e. we have significant build time reduction on a weak machine, if we use Ninja instead of GNU Make.